k8s/clusterrole.go

```diff
@@ -0,0 +1,28 @@
+package k8s
+
+import (
+	"bytes"
+
+	rbacv1 "k8s.io/api/rbac/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	yamlutil "k8s.io/apimachinery/pkg/util/yaml"
+	"k8s.io/client-go/kubernetes"
+)
+
+func UpdateClusterRoleBindingFromYaml(k8sClient *kubernetes.Clientset, clusterRoleBindingYaml string) error {
+	clusterRoleBinding := rbacv1.ClusterRoleBinding{}
+	decoder := yamlutil.NewYAMLToJSONDecoder(bytes.NewReader([]byte(clusterRoleBindingYaml)))
+	if err := decoder.Decode(&clusterRoleBinding); err != nil {
+		return err
+	}
+	if _, err := k8sClient.RbacV1().ClusterRoleBindings().Create(&clusterRoleBinding); err != nil {
+		if apierrors.IsAlreadyExists(err) {
+			if _, err := k8sClient.RbacV1().ClusterRoleBindings().Update(&clusterRoleBinding); err != nil {
+				return err
```

Comment:
We should add retry for N seconds to make sure kubernetes is up and running, this causes errors in newly created clusters 
```
FATA[0013] Failed to apply the ServiceAccount needed for job execution: Post https://35.226.36.97:6443/apis/rbac.authorization.k8s.io/v1/clusterrolebindings: dial tcp 35.226.36.97:6443: getsockopt: connection refused  
```